docs/how_to/create_custom_coords.rst

```diff
@@ -0,0 +1,38 @@
+.. _sunpy-how-to-create-custom-coordinate-objects:
+
+**********************************************
+Create Coordinate Objects with Custom Observer
+**********************************************
+
+It is possible to specify an observer other than Earth by providing a `~astropy.coordinates.SkyCoord` object representing the observer's location.
+This can be useful when working with data from instruments located at specific points in space, such as spacecraft.
+
+.. code-block:: python
+
+    >>> from astropy.coordinates import SkyCoord, CartesianRepresentation
+    >>> from sunpy.coordinates import frames
+    >>> import astropy.units as u
+
+Define the observer's position in Heliocentric coordinates
+
+.. code-block:: python
+
+    >>> observer_coord = SkyCoord(-100*u.arcsec, 500*u.arcsec, frame=frames.Helioprojective)
+    >>> observer_coord= SkyCoord(x=-72241.0*u.km, y=361206.1*u.km, z=589951.4*u.km, frame=frames.Heliocentric)
```

Comment:
```suggestion
```
These two lines are bad examples of specifying an observer location.  First, they are both observer-based coordinate frames, so there is an implicit `observer='earth'`.  Second, they are both on the surface of the Sun, which is not where any realistic observer would be.